cli/tauri.js/src/runner.ts

```diff
@@ -162,35 +192,44 @@ class Runner {
             document.head.appendChild(cspTag)
           }
 
-          writeFileSync(path.join(indexDir, 'index.tauri.html'), dom.serialize())
+          writeFileSync(
+            path.join(indexDir, 'index.tauri.html'),
+            dom.serialize()
+          )
           resolve(inlinedAssets)
         }
-      }).on('progress', event => {
+      }).on('progress', (event: string) => {
         const match = event.match(/([\S\d]+)\.([\S\d]+)/g)
         match && inlinedAssets.push(match[0])
       })
     })
   }
 
-  stop () {
+  async stop(): Promise<void> {
     return new Promise((resolve, reject) => {
       this.tauriWatcher && this.tauriWatcher.close()
-      this.__stopCargo().then(resolve)
+      this.__stopCargo()
+        .then(resolve)
+        .catch(e => {
+          console.error(e)
+        })
     })
   }
 
-  __runCargoCommand ({
+  async __runCargoCommand({
```

Comment:
I think this method will need some refactor: why are we using `async` and a promise, here? It doesn't look useful at all since we're resolving right after the `spawn` call.

Reply:
There is a recommended linter rule that says all functions that return promises should be async. We can disable that if you prefer, but I kinda like it.

Reply:
I guess it's for the sake of explicitness. I see nothing harmful keeping that rule, then. I'm just gonna update my own firmware, haha. 
Nevertheless, the `async` keyword is not the only problem, here. Anyway, it's out of this PR scope and not blocking.